### PR TITLE
Some fine-tuning for the phpunit.xml.dist

### PR DESCRIPTION
**Pull request by OskarStark**
After merging #370 

I encountered some small things.

- [x] move the `<listener>` below `<filter>`
- [ ] change the test-suite name ([see my comment](https://github.com/sonata-project/SonataTranslationBundle/pull/232/files#r191992072))

### CODE REVIEW

**Comment by covex-nn** [comment]
@OskarStark a buid https://travis-ci.org/sonata-project/SonataDashboardBundle/builds/386024795 was failed with `Class "Symfony\Bridge\PhpUnit\SymfonyTestsListener" does not exist`. Is is related to #370?

**Comment by OskarStark** [comment]
But dev-kit merged the branch and all checks pass...

https://github.com/sonata-project/SonataDashboardBundle/pull/143

any idea @greg0ire @Soullivaneuh ?

**Comment by greg0ire** [comment]
Install the bridge?

**Comment by covex-nn** [comment]
There was no `symfony/phpunit-bridge` in composer.json =) I added bridge in https://github.com/sonata-project/SonataDashboardBundle/pull/144, travis is green

**Comment by OskarStark** [comment]
@Soullivaneuh please give me feedback here, thank you!

**Comment by OskarStark** [comment]
@Soullivaneuh can you please give me feedback?

**Comment by OskarStark** [comment]
Can anyone else help me?
I would like to finish this PR.

File: project/phpunit.xml.dist [hunk 1]
```diff
@@ -23,16 +23,16 @@ It's auto-generated by sonata-project/dev-kit package.
...
-    </listeners>
-
     <filter>
         <whitelist>
             <directory suffix=".php">./src/</directory>
         </whitelist>
     </filter>
 
+    <listeners>
+        <listener class="Symfony\Bridge\PhpUnit\SymfonyTestsListener"/>
```
Comment (Soullivaneuh) [inline project/phpunit.xml.dist]: Well. It is just a move?

Comment (OskarStark) [reply project/phpunit.xml.dist]: Yes I have an open Todo and wanted to have some feedback :)

**Pull request closed without merging**